packages/component/src/SendBox/SendButton.js

```diff
@@ -14,16 +14,21 @@ const connectSendButton = (...selectors) =>
         submitSendBox();
       },
       disabled,
-      language
+      language,
+      sendBoxValue
     }),
     ...selectors
   );
 
-const SendButton = ({ click, disabled, language }) => (
-  <IconButton alt={localize('Send', language)} disabled={disabled} onClick={click}>
-    <SendIcon />
-  </IconButton>
-);
+const SendButton = ({ click, disabled, language, sendBoxValue }) => {
+  disabled = sendBoxValue ? false : true;
```

Comment:
My concern is that this is overwriting the props value `disabled`. @compulim, thoughts?

Reply:
@corinagum : Here are snapshots of the issue which i have fixed

![image](https://user-images.githubusercontent.com/3982352/61765813-40adb300-adfc-11e9-92a0-b336e5f67be2.png)

![image](https://user-images.githubusercontent.com/3982352/61765818-43a8a380-adfc-11e9-912d-ad9741aaba95.png)